intelmqmail/cb.py

```diff
@@ -856,3 +856,7 @@ def mailgen(args, config):
     os.environ['GNUPGHOME'] = config["openpgp"]["gnupg_home"]
 
     mailgen(args, config)
+
+
+if __name__ == '__main__':
+    main()
```

Comment:
Is it really necessary to be able to execute that module as a script? In my experience it's generally a bad idea to execute a file inside a package as a script because it can lead to confusing errors because the package directory itself is added to sys.path.